schemas/schema-schema.ipldsch

```diff
@@ -361,6 +405,9 @@ type InlineDefn union {
 type StructRepresentation union {
 	| StructRepresentation_Map "map"
 	| StructRepresentation_Tuple "tuple"
+	| Representation_StringPairs "stringpairs"
+	| StructRepresentation_StringJoin "stringjoin"
+	| Representation_listpairs "listpairs"
```

Comment:
Temptation: to consider a 'type alias' feature.

I dunno if I wanna broach that at the moment though without putting in some work to come up with more examples of what that could be like.

Reply:
Coming back to this with fresh eyes: I think I *would* rather "duplicate" things (e.g. have StructRepresentation_StringPairs and also MapRepresentation_StringPairs).

Reason: it's "coincidental" (relatively) that similar strategies exist for representing maps and structs.  The fact that structurally similar configurations are required for them presently doesn't indicate they should be coupled.